autopilot/contractor.go

```diff
@@ -51,6 +43,12 @@ type (
 		mu         sync.Mutex
 		currPeriod uint64
 	}
+
+	contractInfo struct {
```

Comment:
I considered extending and renaming `api.Contract` but haven't for the time being. It wouldn't be a bad idea though I think.. wdyt?